vtune/README.md

```diff
@@ -0,0 +1,81 @@
+## VTune analysis
+
+First, the code has been run under VTune on DevCloud "cluster" and we obtained the following picture:  
```

Comment:
Ok. I think this introduction was too tough and direct. We can add:

* Why is VTune intended for?
* Which are the technical specs of the cluster?
* How was it run: command?
* Which was the setup of the run: how many cores per node, how many nodes, SIMD capabilities...?